server/views/pages/index.njk

```diff
@@ -81,32 +81,35 @@
         {% if showBookerManagementTile %}
           <li class="govuk-grid-column-one-third card-group__item">
             {{ card({
-              "href": "/manage-bookers/search",
-              "clickable": "true",
-              "heading": "Manage online bookers",
-              "description": "Link visitors to public booker accounts",
-              "id": "booker-management"
+              href: "/manage-bookers/search",
+              clickable: "true",
+              heading:
+                '<span class="bapv-nowrap">Manage online bookers' +
+                  notificationCountBadge(visitorRequestCount, "visitor-request-count") +
+                "</span>",
```

Comment:
This is the actual relevant change in this file: other changes are just tidying up unnecessary quoting of macro parameters.

Reply:
I see why though